modules/social_features/social_landing_page/config/install/core.entity_view_display.profile.profile.featured.yml

```diff
@@ -16,7 +16,7 @@ dependencies:
     - field.field.profile.profile.field_profile_profile_tag
     - field.field.profile.profile.field_profile_self_introduction
     - field.field.profile.profile.field_profile_show_email
-    - image.style.social_featured
+    - image.style.social_large
```

Comment:
The `social_featured` image style was created for this view mode, so if we need to change it because we're changing how landing pages look then we should probably just change the image style itself.

Additionally this will also impact "old-style" landing pages.

Reply:
We need to change image style on featured teasers only for profiles (for both old and sky - this was agreed with Xinyu).
But we still use `social_featured` image style for featured content and groups.
So in this way, it makes the most sense to change image style to ones that fit with a new design.
Also, it shouldn't break anything, since CSS for both old and new styles were updated as well